src/Notify/Notifier/AbstractFcmNotifier.php

```diff
@@ -96,7 +100,7 @@ protected function buildPayload(Message $notification, Message $app, ?Message $c
                 'body' => $notification->get('body', $title),
             ],
             'fcm_options'  => [
-                'analytics_label' => $notification->get('_id')->toString(),
+                'analytics_label' => $this->generateAnalyticsLabel($notification),
```

Comment:
@bhsiao55 pass the notification and the app to the generateAnalyticsLabel.  just in case at the site level you want to customize it based on the app.